app/lib/irs1040_pdf.rb

```diff
@@ -30,11 +31,12 @@ def hash_for_pdf
         AdjustedGrossIncomeAmt11: 1,
         TotalItemizedOrStandardDedAmt12a: @tax_return.standard_deduction,
         TaxableIncomeAmt15: 0,
+        AdditionalChildTaxCreditAmt28: @benefits.outstanding_ctc_amount,
         RecoveryRebateCreditAmt30: @benefits.claimed_recovery_rebate_credit,
-        RefundableCreditsAmt32: @benefits.claimed_recovery_rebate_credit,
-        TotalPaymentsAmt33: @benefits.claimed_recovery_rebate_credit,
-        OverpaidAmt34: @benefits.claimed_recovery_rebate_credit,
-        RefundAmt35: @benefits.claimed_recovery_rebate_credit,
+        RefundableCreditsAmt32: refundable_credits,
```

Comment:
since these values are calculated on both the PDF and on the XML itself, it likely makes sense to create a calculated value on the benefits object called 'total_refundable_credits'.